shell/src/main/java/org/crsh/util/JNDIHandler.java

```diff
@@ -11,7 +11,11 @@
 import java.util.regex.Pattern;
 
 /** @author <a href="mailto:alain.defrance@exoplatform.com">Alain Defrance</a> */
-public class JNDIHandler {
+public final class JNDIHandler {
+	
+  private JNDIHandler(){
+	  throw new AssertionError("Must not instantiate this class");
```

Comment:
If this is really necessary, perhaps it should throw an IllegalStateException, instead.